interface/forms/soap/templates/general_new.html

```diff
@@ -4,49 +4,79 @@
 <title>{xl t='SOAP'|escape:'html'}</title>
 
 {headerTemplate}
+{literal}
+ <style type="text/css" title="mystyles" media="all">
+ <!--
```

Comment:
I agree that internal stylesheets take away the main advantages of the external stylesheets. Removing this stylesheet at this juncture will revert styling of these pages to the default bootstrap style.  The most noticeable effect would be that it would break the style of display of blocks of elements in the form as the base bootstrap does not assign separate  background colors to legends and fieldsets. 
It is vital for the fieldsets and legends to be distinguishable from the background as all the forms that have been so styled break up the form into logical sections using fieldsets and legends that, in my opinion, makes for an easier user experience.
To see what I am talking about use light.css, inspect the page in chrome/firefox and delete this internal stylesheet.  Of course this will not affect flat red as I have included these styles in that external stylesheet ;)
 I have carefully styled the various elements in flat red to minimize the difference in display of various elements in pages styled using bootstrap and those without. Once everything gets converted to bootstrap the need for such overrides it will diminish.
These internal styles will likely end up in an external stylesheet that can be imported into the other external stylesheets  à la core.css. 
One thing you would have noticed in trying to make the forms responsive we need to use media queries. These  need to be individualized for the various forms and would likely require internal stylesheets.